src/utils/Xhr.js

```diff
@@ -118,10 +126,17 @@ class Xhr {
             return false;
         }
 
+        // Network methods that are safe to retry
+        const idempotentMethods = [HTTP_GET, HTTP_OPTIONS, HTTP_HEAD];
+
         const { response, request } = error;
         // Retry if there is a network error (e.g. ECONNRESET) or rate limited
+        const status = getProp(response, 'status');
+        const method = getProp(request, 'method');
```

Comment:
TODO: confirm the HTTP method is actually available on the Axios request object

Reply:
https://github.com/axios/axios/blob/master/lib/core/Axios.js#L38 `request.config.method`?